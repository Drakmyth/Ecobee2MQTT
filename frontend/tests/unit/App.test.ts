import { shallowMount } from '@vue/test-utils';
import App from '@/App.vue';

describe('App.vue', () => {
    test('renders correctly', () => {
        const wrapper = shallowMount(App);
        expect(wrapper.html()).toMatchSnapshot();
    });
});