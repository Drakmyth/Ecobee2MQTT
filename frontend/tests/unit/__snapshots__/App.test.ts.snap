// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue renders correctly 1`] = `
"<header>
  <nav-bar-stub></nav-bar-stub>
</header>
<router-view-stub name=\\"default\\"></router-view-stub>"
`;
